internal/kubernetes/operator/features/crds.go

```diff
@@ -120,7 +130,7 @@ func CRDCompatibleVersion(operatorVersion string) (string, error) {
 	return semver.New(
 		operatorVersionSem.Major(),
 		operatorVersionSem.Minor(),
-		0,
+		operatorVersionSem.Patch(),
```

Comment:
I am not happy about this @helderjs @igor-karpukhin: I think we should declare this a 2.2 release and not 2.1.1.